phoenix-core/src/it/java/org/apache/phoenix/end2end/CDCMiscIT.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.phoenix.end2end;
+
+import org.apache.phoenix.exception.SQLExceptionCode;
+import org.apache.phoenix.query.BaseTest;
+import org.apache.phoenix.util.SchemaUtil;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.Properties;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.fail;
+
+@Category(ParallelStatsDisabledTest.class)
+public class CDCMiscIT extends ParallelStatsDisabledIT {
+    private void assertCDCState(Connection conn, String cdcName, String expInclude,
+                                int idxType) throws SQLException {
+        try (ResultSet rs = conn.createStatement().executeQuery("SELECT cdc_include FROM " +
```

Comment:
Once all the SELECT queries to SYSTEM.CATALOG are replaced with PTable based comparison, we are good to go

Reply:
I have added the check for the change scope on `PTable` as it corresponds to the protobuf change but left the existing SYSCAT checks as is, as these are existing fields and don't necessarily have to be validated through `PTable`. WDYT?